src/UnitTests/Gui/Windows/Forms/MainFormInteractorTests.cs

```diff
@@ -246,7 +247,10 @@ public void Mfi_Save()
     </user>
   </input>
 </project>";
-            Assert.AreEqual(s, interactor.Test_SavedProjectXml);
+            // xml namespaces appear in different order on different platforms, so remove
```

Comment:
The way we've solved this in the past is to use the `XmlnsHidingWriter` which strips the xmlns attributes in a derived class of `XmlWriter`. However, the `MainFormInteractor` doesn't expose a way to override the creation of an `XmlWriter` -- yet. I will accept this for now, and then rewrite `MainFormInteractor` to start using `XmlTextWriter` explicitly.